doc/tables/code-managers.csv

```diff
@@ -0,0 +1,20 @@
+Affiliation;Code Manager;Areas of Expertise
+EPIC;**Michael Lueken (@MichaelLueken)**;Lead SRW App code manager
+GSL;Daniel Abdi (@danielabdi-noaa);Workflow generation, testing RRFS on the cloud, environment modules  
+GSL;Jeff Beck (@JeffBeck-NOAA);SRW App configuration/workflow, code management, meteorological evaluation  
+EMC;Ben Blake (@BenjaminBlake-NOAA);Output visualization, Rocoto  
+GSL;Christopher Harrop (@christopherwharrop-noaa);Rocoto, code management, and testing  
+GSL;Christina Holt (@christinaholtNOAA);Workflow, conda environment support, testing, and code management  
+EPIC;Chan-Hoo Jeon (@chan-hoo);Air quality modeling (Online-CMAQ), Operational platform testing (WCOSS/NCO), Workflow  
```

Comment:
@chan-hoo - Now that you are with EPIC, are the `Areas of Expertise` still correct, or would you like to make modifications?